packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -260,16 +288,28 @@ class SkiaGoldClient {
   /// Executes the `imgtest init` command in the goldctl tool for tryjobs.
   ///
   /// The `imgtest` command collects and uploads test results to the Skia Gold
-  /// backend, the `init` argument initializes the current tryjob.
+  /// backend, the `init` argument initializes the current tryjob. Used by the
+  /// [_AuthorizedFlutterPreSubmitComparator].
   Future<void> tryjobInit() async {
     final File keys = workDirectory.childFile('keys.json');
     final File failures = workDirectory.childFile('failures.json');
 
     await keys.writeAsString(_getKeysJSON());
     await failures.create();
     final String commitHash = await _getCurrentCommit();
-    final String pullRequest = platform.environment['CIRRUS_PR'];
-    final String cirrusTaskID = platform.environment['CIRRUS_TASK_ID'];
+
+    String pullRequest;
+    String jobId;
+    String cis = ci;
+    if (ci == 'luci') {
+      jobId = platform.environment['LOGDOG_STREAM_PREFIX'].split('/').last;
```

Comment:
Is this splitting covered by a unit test somewhere?

Reply:
It is now! :)